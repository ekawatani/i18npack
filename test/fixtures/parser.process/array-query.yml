foos:
  - value: foo_text1
  - value: foo_text2
bar: "{{ foos[1].value }}"
