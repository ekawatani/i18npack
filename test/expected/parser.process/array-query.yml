foos:
  - value: foo_text1
  - value: foo_text2
bar: foo_text2
